src/SignUpIn/SignIn.js

```diff
@@ -1,16 +1,24 @@
 // No log in functionality yet
```

Comment:
remove this comment since you added log in

Reply:
Done